docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.8"
+services:
+  webapi:
+    build:
+      context: .
+    command: /bin/bash -c "./docker-entrypoint.sh dotnet Dfe.Academies.Academisation.WebApi.dll"
+    ports:
+      - "80:80"
+    depends_on:
+      - db
+    env_file:
+       - .env.development.local
+    networks:
+      - dev
+
+  db:
```

Comment:
Maybe we should be using - ghcr.io/dfe-digital/trams-development-database:latest here instead?

Reply:
Is that a custom mssql image for this service?
I was attempting to have the docker image be able to run against a standard MSSQL instance

Reply:
So as part of this pipeline you were intending to provision a new sql database instance? If so that would make it a proper microservice so I'm not against it 🙂
Though this might have an implication on how easy it will be to migrate the data from the existing db. I know Wystan had intended we use the existing db, with a new schema. Tagging @kolevk for visibility.

Yes, its the academies database schema, though now I understand a bit better perhaps not suitable to be used here.

Reply:
shouldn't be running against a new SQL server. Should be consuming existing test database

Reply:
or live

Reply:
I've since caught up with @Stretch96 on this. The docker compose is intended for local development & the CI server. There are no plans to provision a new DB instance. It was my misunderstanding.

Reply:
This is just here for local/CI testing - This file isn't used to provision live infrastructure

Reply:
bit we don;t need / use a docker SQL server now, so why do we need one?

Reply:
The webapi container can't launch without an SQL database
So to run the webapi container locally, it needs an SQL database